CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [6.1.5](https://github.com/cleandart/react-dart/compare/6.1.4...6.1.5)##
+- [#333] Remove old, unused file from bin/
```

Comment:
#nit 
```suggestion
- [#333] Remove old, unused file from bin.
```

Reply:
I actually did a slash on purpose to indicate it was a directory

Reply:
oh. well i did not get that.